app/src/main/java/com/taskruler/dto/User.kt

```diff
@@ -3,4 +3,4 @@ package com.taskruler.dto
 /**
  * Data class to setup unique users for app usage/authentication/Firebase
  */
-data class User(val uid: String = "", var displayName: String? = "")
+data class User(var uid: String = "", var displayName: String? = "")
```

Comment:
Here the parameter **uid** is modified after being initialized as "". Here it should be **var**